File: src/frontend/src/app/services/Dtos/DraftRequestDto.ts
export class DraftRequestDto {
    url: string;
    picture: string;
    title: string;
    paragraphs: string[];
    tags: string[];

    constructor() {
        this.url = '';
        this.picture = '';
        this.title = '';
        this.paragraphs = [];
        this.tags = [];
    }
}

File: src/backend/NewsTrack.WebApi/Dtos/DraftRequestDto.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace NewsTrack.WebApi.Dtos
{
    public class DraftRequestDto
    {
        [Url]
        [Required]
        public string Url { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public IEnumerable<string> Paragraphs { get; set; }

        [Url]
        [Required]
        public string Picture { get; set; }

        public IEnumerable<string> Tags { get; set; }
    }
}
